api/resources/get_transaction.py

```diff
@@ -0,0 +1,36 @@
+from flask_restful import Resource
+from flask import request
+from config import app
+import iota
+import json
+
+
+class Get_transaction (Resource):
+    def get(self, name):
```

Comment:
Do you think `tx_hash` is better than `name` ?

Reply:
Fix in 629e94e.